### PR TITLE
Feature/removeButton

### PR DESCRIPTION
**Pull request by entotsu**
add remove button feature. :+1:

Please merge if you like this!

### CODE REVIEW

File: TagListView/TagListView.swift [hunk 1]
```diff
@@ -68,18 +69,18 @@ public class TagListView: UIView {
             }
         }
     }
-    @IBInspectable public var paddingY: CGFloat = 2 {
+    @IBInspectable public var paddingX: CGFloat = 2 {
```
Comment (xhacker) [inline TagListView/TagListView.swift]: These changes are redundant, right?

File: TagListView/TagView.swift [hunk 2]
```diff
@@ -101,16 +143,93 @@ public class TagView: UIButton {
...
+            removeButton.frame.size.width = removeButtonSize + paddingX
+            removeButton.frame.origin.x = self.frame.width - removeButton.frame.width
+            removeButton.frame.size.height = self.frame.height
+            removeButton.frame.origin.y = 0
+        }
+    }
 }
+
+
+class CloseButton: UIButton {
```
Comment (xhacker) [inline TagListView/TagView.swift]: Can we merge `DrawCloseView` and `CloseButton` into one class, and make it a separate Swift file?

**Comment by xhacker** [comment]
Hi @entotsu,

Thank you, I like this pr very much! I’ve reviewed and added some small comments. Also, after adding the remove button, the left padding and right padding seem to be unequal. Can you fix that?

screen shot 2016-01-21 at 12 19 51 pm

**Comment by xhacker** [comment]
@entotsu: Just to remind you if you made any changes, please don't forget to @ me. It's been several times that I missed someone's change...

**Comment by entotsu** [comment]
I remember this PR is clean.
I didn't make any changes.
I just add remove button future.
I think merging is good.

**Pull request closed without merging**